nexus/src/external_api/http_entrypoints.rs

```diff
@@ -5967,6 +5975,125 @@ async fn current_user_ssh_key_delete(
     apictx.external_latencies.instrument_dropshot_handler(&rqctx, handler).await
 }
 
+/// List instrumentation probes.
+#[endpoint {
+    method = GET,
+    path = "/v1/probes",
+    tags = ["probes"],
```

Comment:
```suggestion
    tags = ["system/probes"],
```

The `system/` prefix is how we determine the grouping on the docs site.

https://github.com/oxidecomputer/docs/blob/e605c8a871d0e42e6eed4c512b9fcdbeb22f2682/app/routes/api.tsx#L62-L68

Reply:
Done